src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -158,18 +165,26 @@ namespace AppInstaller::CLI::Workflow
         const auto& unsupportedArgs = context.Get<Execution::Data::Installer>()->UnsupportedArguments;
         for (auto unsupportedArg : unsupportedArgs)
         {
-            const auto& argument = GetUnsupportedArgument(unsupportedArg);
-            if (context.Args.Contains(argument.ExecArgType()))
+            const auto& unsupportedArgType = GetUnsupportedArgumentType(unsupportedArg);
+            if (context.Args.Contains(unsupportedArgType))
             {
-                const auto& usageString = argument.GetUsageString();
-                if (ShouldErrorForUnsupportedArgument(unsupportedArg))
-                {
-                    context.Reporter.Error() << Resource::String::UnsupportedArgument << ' ' <<  usageString << std::endl;
-                    AICLI_TERMINATE_CONTEXT(APPINSTALLER_CLI_ERROR_UNSUPPORTED_ARGUMENT);
-                }
-                else
+                const auto& commandArguments = context.GetExecutingCommand()->GetArguments();

```

Comment:
You cannot assume that the executing command has been set.  I would make sure that `Resource::String::UnsupportedArgument` made sense if alone, then if the command is null just don't have a usage string after.

Reply:
Changed `UnsupportedArgument` string so that it makes sense alone. Checks to make sure command is not null before attempting to print usage strings.